aircraft/cost.py

```diff
@@ -0,0 +1,112 @@
+"""Aircraft cost models"""
+from gpkit.shortcuts import Var, Model
+import gpkit
+gpkit.disable_units()
```

Comment:
Did we decide on a convention for now? `$/hr` verses `1/hr`? And if we decide to go with `1/hr` should I get rid of the `gpkit.disable_units()`?